src/flake8_to_ruff/converter.rs

```diff
@@ -65,11 +65,17 @@ pub fn convert(
     let plugins = plugins.unwrap_or_else(|| {
         let from_options = plugin::infer_plugins_from_options(flake8);
         if !from_options.is_empty() {
-            eprintln!("Inferred plugins from settings: {from_options:#?}");
+            #[allow(clippy::print_stderr)]
+            {
+                eprintln!("Inferred plugins from settings: {from_options:#?}");
```

Comment:
Obviously a little absurd to add these everywhere, but I'd really like to be defensive about this stuff, and we print so rarely that it doesn't bother me to have to guard like this. (The `{` and `}` is required since otherwise the macro gets expanded and the `allow` is ignored.)